yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -1309,17 +1310,28 @@ def pixelize_sph_kernel_gather_arbitrary_grid(np.float64_t[:, :, :] buff,
     kernel_func = get_kernel_func(kernel_name)
 
     # generate the kdtree and find neighbours
+    # I think there is a better way of the passing than making a new array
```

Comment:
@ngoldbaum can you recommend a better way of passing these memory views as a multi dimensional numpy array? i think this process is very slow!